addon/-private/system/references.js

```diff
@@ -1,5 +1,5 @@
-import RecordReference from './references/record';
-import BelongsToReference from './references/belongs-to';
-import HasManyReference from './references/has-many';
+import RecordReference from 'ember-data/-private/system/references/record';
```

Comment:
relative should work correctly, if it does not that is a bug in the build we need to fix.